frontend/src/scenes/ingestion/v2/IngestionWizard.tsx

```diff
@@ -25,14 +25,27 @@ import { NoDemoIngestionPanel } from './panels/NoDemoIngestionPanel'
 
 export function IngestionWizardV2(): JSX.Element {
     const { currentView, platform } = useValues(ingestionLogicV2)
-    const { reportIngestionLandingSeen } = useActions(eventUsageLogic)
+    const { reportIngestionLandingSeen, reportIngestionWait } = useActions(eventUsageLogic)
+
+    const [minutesPassed, setMinutesPassed] = useState(0)
+    const timeout = useRef(0)
```

Comment:
Should this be in kea? (that's an actual question, not a directive, and only because I know there's a "kea-not-local-state" preference here, but sometimes it does feel a bit... heavy)